docs/configuration.asciidoc

```diff
@@ -104,6 +104,31 @@ class MyApp extends android.app.Application {
 }
 ----
 
+[float]
+[[server-export-protocol]]
+==== APM Server export protocol
+
+The signals will be exported using the gRPC protocol by default, you could change it to HTTP using the ExportProtocol configuration as shown below:
+
+[source,java]
+----
+class MyApp extends android.app.Application {
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        ElasticApmConfiguration configuration = ElasticApmConfiguration.builder()
+                .setExportProtocol(ExportProtocol.HTTP) <1>
+                .build();
+        ElasticApmAgent.initialize(this, configuration);
+    }
+}
+----
+
+<1> Note that this configuration may not work if a <<custom-signal-configuration, custom signal configuration>> is set.
+
+NOTE: OTLP over HTTP support was added to the APM server on its version https://www.elastic.co/guide/en/apm/guide/current/release-notes-8.3.html#_added_8[8.3.0], whereas OTLP over gRPC is supported since version https://www.elastic.co/guide/en/apm/server/current/release-notes-7.12.html#_added_5[7.12.0].
```

Comment:
These are really old versions, so I don't think we need to link to the release notes here. I'd remove the links and just add the supported version.

```suggestion
NOTE: OTLP over HTTP is supported in APM server versions 8.3.0+. OTLP over gRPC is supported in APM Server versions 7.12.0+.
```